README.md

```diff
@@ -1,85 +1,93 @@
 Stylo
 =====
 
-This repo contains Servo’s downstream fork of [Stylo](https://searchfox.org/mozilla-central/source/servo).
+**High-performance CSS style engine**
 
-The branches are as follows:
+[![Build Status](https://github.com/servo/stylo/actions/workflows/main.yml/badge.svg)](https://github.com/servo/stylo/actions)
+[![Crates.io](https://img.shields.io/crates/v/stylo.svg)](https://crates.io/crates/stylo)
+[![Docs](https://docs.rs/stylo/badge.svg)](https://docs.rs/stylo)
+![Crates.io License](https://img.shields.io/crates/l/stylo)
 
-- [`upstream`](https://github.com/servo/style/tree/upstream) has upstream mozilla-central filtered to the paths we care about ([style.paths](style.paths)), but is otherwise unmodified
-- [`main`](https://github.com/servo/style/tree/ci) has our downstream patches, plus the scripts and workflows for syncing with mozilla-central, to be rebased onto `upstream`
+Stylo is a high-performance, browser-grade CSS style engine written in Rust that powers [Servo](https://servo.org) and [Firefox](https://firefox.com). This repo contains Servo’s downstream version of Stylo. The upstream version lives in mozilla-central with the rest of the Gecko/Firefox codebase.
 
-## Building Servo against your local Stylo
+Coordination of Stylo development happens:
 
-Assuming your local `servo` and `stylo` directories are siblings, you can build `servo` against `stylo` by adding the following to `servo/Cargo.toml`:
+- Here in Github Issues
+- In the [#stylo](https://servo.zulipchat.com/#narrow/channel/417109-stylo) channel of the [Servo Zulip](https://servo.zulipchat.com/)
+- In the [#layout](https://chat.mozilla.org/#/room/#layout:mozilla.org) room of the Mozilla Matrix instance (matrix.mozilla.org)
 
-```toml
-[patch."https://github.com/servo/stylo"]
-selectors = { path = "../stylo/selectors" }
-servo_arc = { path = "../stylo/servo_arc" }
-stylo_atoms = { path = "../stylo/stylo_atoms" }
-stylo = { path = "../stylo/style" }
-stylo_config = { path = "../stylo/stylo_config" }
-stylo_dom = { path = "../stylo/stylo_dom" }
-stylo_malloc_size_of = { path = "../stylo/malloc_size_of" }
-stylo_traits = { path = "../stylo/style_traits" }
-```
+## High-level Documentation
```

Comment:
Nit: Perhaps you can use Title Case for all headings or none of them. See also below "Stylo Crates" and "Standalone crates."